tests/unit/amo/api/test_collections.js

```diff
@@ -0,0 +1,159 @@
+import * as api from 'core/api';
+import {
+  getCollectionAddons,
+  getCollectionDetail,
+} from 'amo/api/collections';
+import { parsePage } from 'core/utils';
+import { createApiResponse } from 'tests/unit/helpers';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+} from 'tests/unit/amo/helpers';
+
+
+describe(__filename, () => {
+  let mockApi;
+
+  const getParams = ({
+    apiState = dispatchClientMetadata().store.getState().api,
```

Comment:
Since you reference in a few places where it's not relevant to the test, you might as well initialize it in `beforeEach()`